test/e2e/apps/statefulset.go

```diff
@@ -882,6 +889,89 @@ var _ = SIGDescribe("StatefulSet", func() {
 			appTester.run()
 		})
 	})
+
+	framework.KubeDescribe("V1beta2 StatefulSet Scale Subresource [Feature:StatefulSet]", func() {
+		ssName := "ss"
+		labels := map[string]string{
+			"foo": "bar",
+			"baz": "blah",
+		}
+		headlessSvcName := "test"
+		var statefulPodMounts, podMounts []v1.VolumeMount
+		var ss *appsV1beta2.StatefulSet
+
+		BeforeEach(func() {
+			statefulPodMounts = []v1.VolumeMount{{Name: "datadir", MountPath: "/data/"}}
+			podMounts = []v1.VolumeMount{{Name: "home", MountPath: "/home"}}
+			ss = framework.NewStatefulSetV1beta2(ssName, ns, headlessSvcName, 2, statefulPodMounts, podMounts, labels)
+
+			By("Creating service " + headlessSvcName + " in namespace " + ns)
+			headlessService := framework.CreateServiceSpec(headlessSvcName, "", true, labels)
+			_, err := c.Core().Services(ns).Create(headlessService)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		AfterEach(func() {
+			if CurrentGinkgoTestDescription().Failed {
+				framework.DumpDebugInfo(c, ns)
+			}
+			framework.Logf("Deleting all statefulset in ns %v", ns)
+			framework.DeleteAllStatefulSets(c, ns)
+		})
+
+		It("should be modifiable", func() {
+			By("Creating statefulset " + ssName + " in namespace " + ns)
+			*(ss.Spec.Replicas) = 1
+			sst := framework.NewStatefulSetTester(c)
+			sst.PauseNewPodsV1beta2(ss)
+
+			_, err := c.AppsV1beta2().StatefulSets(ns).Create(ss)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("starting proxy server")
```

Comment:
Please add a TODO here explaining that we should use a real client instead of proxy+curl, once we have one that works for StatefulSet Scale subresource.